File: atcoder/2HC2019/zip/toolkitB/scripts/sample_code/sample.py
import sys
# recieve |V| and |E|
V, E = map(int, input().split())
es = [[] for i in range(V)]

for i in range(E):

	# recieve edges
	a, b, c = map(int, input().split())
	a, b = a-1, b-1
	es[a].append((b,c))
	es[b].append((a,c))

# recieve frequencies
F  = list(map(int,input().split()))

T = int(input())

# recieve info
for i in range(T):

	# the number of new order
	Nnew = int(input())
	for j in range(Nnew):
		new_id, dst = map(int, input().split())
	
	# the number of items transfered into the car at time t.
	Nput = int(input())
	for j in range(Nput):
		put_id = int(input())

	# insert your code here to get more meaningful output
	# you have to output a command here
	print(-1)

	vardict = input()
	if vardict == 'NG':
		sys.exit()

	# the number of orders that have been delivered at time t.
	Nachive = int(input())
	for j in range(Nachive):
		achive_id = int(input())

File: atcoder/2HC2019/zip/toolkitB/scripts/sample_code/sample.cpp
#include <iostream>
#include <vector>
#include <utility>
#include <string>

int main() {
		
	// recieve |V| and |E|
	int V, E; std::cin >> V >> E;

	// adjacency list
	std::vector<std::vector<std::pair<int, int>>> G(V);

	for(int i = 0; i < E; i++) {
		// recieve edges
		int u, v, d; std::cin >> u >> v >> d;
		u--; v--;
		G[u].push_back({v, d});
		G[v].push_back({u, d});
	}
	
	// recieve freqencies
	std::vector<int> freq(V);
	for(int i = 0; i < V; i++) std::cin >> freq[i];
	
	int Tmax; std::cin >> Tmax;

	for(int t = 0; t < Tmax; t++) {

		// the number of new order
		int Nnew; std::cin >> Nnew;
		for(int j = 0; j < Nnew; j++) {
			int new_id, dst; std::cin >> new_id >> dst;
		}
	
		// the number of items transfered into the car at time t.
		int Nput; std::cin >> Nput;
		for(int j = 0; j < Nput; j++) {
			int put_id; std::cin >> put_id;
		}

    // insert your code here to get more meaningful output
		// you have to output a command here
		std::cout << -1 << std::endl; // stay

		std::string verdict; std::cin >> verdict;
		if(verdict == "NG") {
			return 1;
		}

		// the number of orders that have been delivered at time t.
		int Nachive; std::cin >> Nachive;
		for(int j = 0; j < Nachive; j++) {
			int achive_id; std::cin >> achive_id;
		}
		
	}

	return 0;
}
